contributing/getting-started-e2e.md

```diff
@@ -1,3 +1,19 @@
+<!---
```

Comment:
Just a note: We are inconsistent (because it makes little sense), but our rules do not require this header on MD files. For checked in code a presubmit enforces that files are properly annotated with a license reference.